pkg/cloud/gke/gke.go

```diff
@@ -90,13 +92,16 @@ func stringFromValue(v starlark.Value) (string, error) {
 	return string(s), nil
 }
 
-func clpFromClusterCtx(c *addon.SkyCtx) (cluster, location, project string, err error) {
+func clpFromClusterCtx(c *addon.SkyCtx) (cluster, location, project, useInternalIP string, err error) {
 	if cluster, err = stringFromValue(c.Attrs[ClusterKey]); err != nil {
 		return
 	}
 	if location, err = stringFromValue(c.Attrs[LocationKey]); err != nil {
 		return
 	}
-	project, err = stringFromValue(c.Attrs[ProjectKey])
+	if project, err = stringFromValue(c.Attrs[ProjectKey]); err != nil {
+		return
+	}
+	useInternalIP, _ = stringFromValue(c.Attrs[UseInternalIPKey])
```

Comment:
It's not required, but still wanna reuse this function, so thinking not return error when the key-value is not found.

Reply:
but ignoring errors generally is an anti-pattern. Maybe just replace this with
```go
useInternalIP := c.Attrs[UseInternalIPKey].(starlark.String)
```

Reply:
ahh, but if `c.Attrs[UseInternalIPKey]` is `nil`, it'd probably crash? let me try

Reply:
I guess I will add those condition check here

Reply:
You are absolutely right about the nil case. I agree we should add the condition check here.

Reply:
updated, ptal, thanks!